### ISSUE TITLE
Virtualenv is not being activated on Windows 10

### ISSUE THREAD

**Issue by nstoik** [pypa/pipenv#1682]
Be sure to check the existing issues (both open and closed!).

Describe the issue briefly here.

When running pipenv in a new folder on Windows 10, the virtualenv and Pipfile are created, but the virtualenv is not used. Running `$ pipenv --venv` shows `No virtualenv has been created for this project yet!`

Any subsequent pipenv commands tries to create another virtualenv. When installing a package like `pipenv install requests`, pipenv tries to install the package globally.

Please run `$ python -m pipenv.help`, and paste the results here.

```
C:\Users\Nelson\temp>python -m pipenv.help

Pipenv version: `'11.5.2'`

Pipenv location: `'C:\\Program Files (x86)\\Python36-32\\lib\\site-packages\\pipenv'`

Python location: `'C:\\Program Files (x86)\\Python36-32\\python.exe'`

Other Python installations in `PATH`:

  - `2.7`: `C:\Python27\python.exe`
  - `3.6`: `C:\Program Files (x86)\Python36-32\python.exe`

  - `3.6.4`: `C:\Program Files (x86)\Python36-32\python.exe`
  - `2.7.12`: `C:\Python27\python.exe`
  - `3.6.4`: `C:\Windows\py.exe`

PEP 508 Information:

{'implementation_name': 'cpython',
 'implementation_version': '3.6.4',
 'os_name': 'nt',
 'platform_machine': 'AMD64',
 'platform_python_implementation': 'CPython',
 'platform_release': '10',
 'platform_system': 'Windows',
 'platform_version': '10.0.17115',
 'python_full_version': '3.6.4',
 'python_version': '3.6',
 'sys_platform': 'win32'}

System environment variables:

  - `ALLUSERSPROFILE`
  - `APPDATA`
  - `COMMONPROGRAMFILES`
  - `COMMONPROGRAMFILES(X86)`
  - `COMMONPROGRAMW6432`
  - `COMPUTERNAME`
  - `COMSPEC`
  - `DRIVERDATA`
  - `FPS_BROWSER_APP_PROFILE_STRING`
  - `FPS_BROWSER_USER_PROFILE_STRING`
  - `HOMEDRIVE`
  - `HOMEPATH`
  - `KICAD_PTEMPLATES`
  - `KISYS3DMOD`
  - `KISYSMOD`
  - `LOCALAPPDATA`
  - `LOGONSERVER`
  - `NUMBER_OF_PROCESSORS`
  - `ONEDRIVE`
  - `OS`
  - `PATH`
  - `PATHEXT`
  - `PROCESSOR_ARCHITECTURE`
  - `PROCESSOR_ARCHITEW6432`
  - `PROCESSOR_IDENTIFIER`
  - `PROCESSOR_LEVEL`
  - `PROCESSOR_REVISION`
  - `PROGRAMDATA`
  - `PROGRAMFILES`
  - `PROGRAMFILES(X86)`
  - `PROGRAMW6432`
  - `PROMPT`
  - `PSMODULEPATH`
  - `PUBLIC`
  - `SESSIONNAME`
  - `SYSTEMDRIVE`
  - `SYSTEMROOT`
  - `TEMP`
  - `TMP`
  - `USERDOMAIN`
  - `USERDOMAIN_ROAMINGPROFILE`
  - `USERNAME`
  - `USERPROFILE`
  - `VS140COMNTOOLS`
  - `WINDIR`
  - `WORKON_HOME`
  - `PIP_PYTHON_PATH`
  - `PYTHONUNBUFFERED`

Pipenv�specific environment variables:


Debug�specific environment variables:

  - `PATH`: `C:\Program Files (x86)\Python36-32\;C:\Program Files (x86)\Python36-32\Scripts\;C:\Python27\;C:\Python27\Scripts;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\Lenovo\Bluetooth Software\;C:\Program Files\Lenovo\Bluetooth Software\syswow64;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;%localappdata%\Microsoft\WindowsApps;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\Client SDK\ODBC\130\Tools\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\Tools\Binn\ManagementStudio\;C:\Program Files\nodejs\;C:\Program Files\dotnet\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Users\Nelson\AppData\Local\Microsoft\WindowsApps;C:\Users\Nelson\AppData\Roaming\npm;C:\Program Files\Microsoft VS Code\bin;C:\Program Files\Heroku\bin;C:\Users\Nelson\AppData\Local\GitHub\PortableGit_f02737a78695063deace08e96d5042710d3e32db\cmd;C:\Users\Nelson\AppData\Local\GitHubDesktop\bin;%USERPROFILE%\AppData\Local\Microsoft\WindowsApps;`
```

---------------------------

If you're on MacOS, just run the following:

    $ python -m pipenv.help | pbcopy

------------

##### Expected result

Describe what you expected.

Run `pipenv shell` or `pipenv install XXX` in a new folder, and have the venv created and start using the venv right away with the package installed in the venv.

##### Actual result

When possible, provide the verbose output (`--verbose`), especially for locking and dependencies resolving issues.
```
C:\Users\Nelson>cd temp

C:\Users\Nelson\temp>pipenv shell --verbose
Creating a virtualenv for this project…
Using c:\program files (x86)\python36-32\python.exe (3.6.4) to create virtualenv…
Already using interpreter c:\program files (x86)\python36-32\python.exe
Using base prefix 'c:\\program files (x86)\\python36-32'
New python executable in E:\Documents\Envs\temp-fJUtMCm1\Scripts\python.exe
Installing setuptools, pip, wheel...done.

Virtualenv location:
Creating a Pipfile for this project…
Launching subshell in virtual environment. Type 'exit' to return.
Microsoft Windows [Version 10.0.17115.1]
(c) 2017 Microsoft Corporation. All rights reserved.

C:\Users\Nelson\temp>exit

C:\Users\Nelson\temp>pipenv --venv
No virtualenv has been created for this project yet!

C:\Users\Nelson\temp>
```

Running with SET PIPENV_VENV_IN_PROJECT=1 gives a different error message but the same result.

```
C:\Users\Nelson\temp>SET PIPENV_VENV_IN_PROJECT=1

C:\Users\Nelson\temp>pipenv shell
Creating a virtualenv for this project…
Using C:\Program Files (x86)\Python36-32\python.exe (3.6.4) to create virtualenv…
Running virtualenv with interpreter C:\Program Files (x86)\Python36-32\python.exe
Using base prefix 'C:\\Program Files (x86)\\Python36-32'
New python executable in C:\Users\Nelson\temp\.venv\Scripts\python.exe
Installing setuptools, pip, wheel...done.

Virtualenv location: C:\Users\Nelson\temp\.venv
ERROR: Environment '.venv' does not exist. Create it with 'pew new .venv'.

C:\Users\Nelson\temp>pipenv --venv
C:\Users\Nelson\temp\.venv

C:\Users\Nelson\temp>pipenv shell
ERROR: Environment '.venv' does not exist. Create it with 'pew new .venv'.

C:\Users\Nelson\temp>
```

##### Steps to replicate

Provide the steps to replicate (which usually at least includes the commands and the Pipfile).

See actual results above.

**Comment by kennethreitz** [comment]
this is strange

**Comment by nstoik** [comment]
this is my first time trying out pipenv (I was using virtualenvwrapper before), so let me know if you need any more information or if I should try anything else.

I tried again just now with version 11.6.1 of pipenv with the same result.

**Comment by techalchemy** [comment]
@kennethreitz did we drop a `normalize_drive` call somewhere...

**Comment by uranusjr** [comment]
Can reproduce. I still need to investigate, but from first glance this is not a `normalize_drive` problem, but the in/out-of project logic is messed up. I’ll try to come up with a fix.

**Comment by techalchemy** [comment]
@uranusjr any idea where this was regressed? See also #1689

**Comment by kennethreitz** [comment]
Thanks @uranusjr!

**Comment by kennethreitz** [comment]
✨🍰✨

**Comment by uranusjr** [comment]
@techalchemy Problem is we used to run a subprocess for `shell` on Windows, but now we run it in-process. I didn’t check when the change was made exactly, but it’s [this line](https://github.com/pypa/pipenv/blob/d7e227f/pipenv/core.py#L2125). It used to be a subprocess call (you can still see the commented out import before it), which would allow the environment variable to be picked up, but now it’s in-process. This combines with some unfortunate design choices on pew’s part results in the bug.

#1689 is a different bug altogether. See #1715.

**Comment by uranusjr** [comment]
Oh and I can confirm this is fixed as of 11.6.7.

**Comment by nstoik** [comment]
its working for me now. thanks for the quick fix!
